target/linux/pistachio/marduk/target.mk

```diff
@@ -25,6 +25,27 @@
 
 BOARDNAME:=marduk
 
+DEFAULT_PACKAGES+=kmod-i2c i2c-tools \
+                  kmod-sound-pistachio-soc alsa-lib alsa-utils alsa-utils-test \
+                  kmod-uccp420wlan kmod-cfg80211 iw hostapd wpa-supplicant \
+                  uboot-envtools tcpdump
+
+define Profile/marduk/default
```

Comment:
We can keep this DEFAULT_PACKAGES  in target.mk since this is what has been followed in various other platforms.